src/main/resources/templates/db_scripts/schema.json

```diff
@@ -167,6 +167,13 @@
           "removeAccents": true
         }
       ],
+      "fullTextIndex": [
+        {
+          "fieldName": "isbn",
```

Comment:
Is this the same name as the index has for items in inventory?

Reply:
You mean in mod-inventory-storage?  If so, yes, the isbn field there is named exactly the same.  Should they be different?

Reply:
> Should they be different?

They could be different. This index is in reference to requests. Do people say, `let's search for requests by ISBN`?

Reply:
Yes it is.

What I'm trying to suggest is that the `ISBN` index for `items` is used for to `search for items with ISBN X`.

Is that the same for requests? Are we trying to `search for requests with ISBN X` or `search for requests for items with ISBN x`?

Reply:
The latter?  (search for requests for items with ISBN x)  So are you suggesting we should rename the index to something like `RequestItemIsbn`?

Reply:
> Are you suggesting we should rename the index to something like `RequestItemIsbn`?

My suggestion would be something like `itemIsbn` given that the index is used in the context of the set of requests, I think the `request` part is redundant

Reply:
Okay, I've changed the indexname in the schema and the tests.